roles/express-api/templates/start-express-container.sh.j2

```diff
@@ -0,0 +1,51 @@
+api_name=$1
+api_path=/apis/apis/$api_name
+env=/apis/env/$api_name.env
+
+cd $api_path
+
+# Stop and destroy container if it's already running
+if [[ "$(docker ps -aq -f name=$api_name)" ]]; then
```

Comment:
We should do some checking before we destroy the container. For example, what if the tar file didn't copy over correctly or doesn't exist? Maybe we should import the new image before destroying the old container?

Reply:
Done.